ledger/participant-integration-api/src/test/lib/scala/platform/store/interning/MockStringInterning.scala

```diff
@@ -65,7 +65,7 @@ class MockStringInterning extends StringInterning {
         rawStringInterning.tryExternalize(id).map(Party.assertFromString)
     }
 
-  private[store] def reset(): Unit = synchronized {
+  def reset(): Unit = synchronized {
```

Comment:
Oh that one rears its ugly head again

Reply:
Ah, it must have reappeared during rebasing/merging and conflict resolution that I was doing. Reverting